doozerlib/cli/release_gen_payload.py

```diff
@@ -587,20 +588,21 @@ def detect_extend_payload_entry_issues(self, assembly_inspector: AssemblyInspect
         self.detect_rhcos_kernel_inconsistencies(targeted_rhcos_builds)
         self.detect_rhcos_inconsistent_rpms(targeted_rhcos_builds)  # across all arches
 
-    def detect_rhcos_issues(self, payload_entry, assembly_inspector: AssemblyInspector):
+    async def detect_rhcos_issues(self, payload_entry: PayloadEntry, assembly_inspector: AssemblyInspector):
         """
         Associate relevant assembly issues with an RHCOS PayloadEntry.
         Use assembly inspector to detect assembly issues like RPMs installed in
         the PayloadEntry that are not what the assembly specifies.
         """
 
         # record issues found in assembly list and per payload entry
-        self.assembly_issues.extend(assembly_inspector.check_rhcos_issues(payload_entry.rhcos_build))
+        rhcos_build = cast(RHCOSBuildInspector, payload_entry.rhcos_build)
+        self.assembly_issues.extend(assembly_inspector.check_rhcos_issues(rhcos_build))
         payload_entry.issues.extend(ai for ai in self.assembly_issues if ai.component == "rhcos")
 
         if self.runtime.assembly_type is AssemblyTypes.STREAM:
             # For stream alone, we want to enforce that the very latest RPMs are installed.
```

Comment:
Not your change, but this comment is misleading. We're not enforcing this, we're emitting a warning. In before times, doozer would allow `OUTDATED_RPMS_IN_STREAM_BUILD` in code. Now we have a `permits` in stream to say we want this to not be a blocker.

Reply:
`enforce` -> `verify`